### PR TITLE
ALP zSeries  deployment

### PR DESCRIPTION
**Pull request by tbazant**

### CODE REVIEW

File: tasks/deploy-alp-zseries.xml [hunk 1]
```diff
@@ -0,0 +1,246 @@
...
+          </para>
+        </note>
+      </step>
+      <step>
+        <para>
+          Optionally, you can use the <command>FILELIST</command> command to
+          list the files and edit the <emphasis role="bold">parmfile</emphasis>
+          with XEDIT. Our example parmfile has the following content:
+        </para>
+<screen>cio_ignore=all,!condev<co xml:id="alp-zseries-cioignore"/>
```
Comment (teclator) [inline tasks/deploy-alp-zseries.xml:167]:
```suggestion
<screen>cio_ignore=all,!condev,!0.0.0160<co xml:id="alp-zseries-cioignore"/>
```

File: tasks/deploy-alp-zseries.xml [hunk 2]
```diff
@@ -0,0 +1,246 @@
...
+rd.cio_accept=0.0.0160
```
Comment (teclator) [inline tasks/deploy-alp-zseries.xml:168]:
```suggestion
```

File: tasks/deploy-alp-dinstaller.xml [hunk 3]
```diff
@@ -133,10 +133,24 @@
...
+            enable switching keyboard layouts, select a password made of
+            alphanumeric characters and be aware of national keyboard layout
+            differences. For extended post-deployment information about disk
+            encryption, refer to
+            <xref linkend="alp-post-deploy-full-disk-encryption"/>.
+          </para>
+        </tip>
+        <para>
+          To configure iSCSI targets or DASD disks (for &zseries;) for the
+          installation, click <guimenu>advanced options menu</guimenu>
```
Comment (dariavladykina) [inline tasks/deploy-alp-dinstaller.xml:153]:
```suggestion
          installation, click <guimenu>advanced options menu</guimenu>.
```

File: tasks/deploy-alp-dinstaller.xml [hunk 4]
```diff
@@ -167,51 +181,36 @@
...
+            <textobject role="description"><phrase>DASD storage contextual menu (&zseries;)</phrase>
             </textobject>
           </mediaobject>
         </figure>
       </step>
+      <step>
+        <para>
+          In the <guimenu>Users</guimenu> section, specify a &rootuser;
+          password, upload a <guimenu>Root SSH public key</guimenu>, or create
+          an additional user account and optionally enable auto-login for it.
```
Comment (dariavladykina) [inline tasks/deploy-alp-dinstaller.xml:202]:
```suggestion
          an additional user account and optionally enable auto login for it.
```

File: tasks/deploy-alp-zseries.xml [hunk 5]
```diff
@@ -0,0 +1,245 @@
...
+  <!ENTITY % entities SYSTEM "../common/generic-entities.ent">
+    %entities;
+]>
+<article xml:id="task-deploy-alp-zseries" xml:lang="en"
+ role="task"
+ xmlns="http://docbook.org/ns/docbook" version="5.1"
+ xmlns:its="http://www.w3.org/2005/11/its"
+ xmlns:xi="http://www.w3.org/2001/XInclude"
+ xmlns:xlink="http://www.w3.org/1999/xlink">
+  <title>Deploying &productnameshort; on a &zseries; host</title>
```
Comment (dariavladykina) [inline tasks/deploy-alp-zseries.xml:18]:
```suggestion
  <title>Deploying &productnameshort; on an &zseries; host</title>
```

File: tasks/deploy-alp-zseries.xml [hunk 6]
```diff
@@ -0,0 +1,245 @@
...
+  <section xml:id="requirements-deploy-alp-zseries">
+    <title>Requirements</title>
+    <para>
+      Before installing &productnameshort; on &zseries;, you need to fullfil
+      the following requirements:
+    </para>
+    <itemizedlist>
+      <listitem>
+        <para>
+          Deployment on the &zseries; architecture is specific and requires
```
Comment (dariavladykina) [inline tasks/deploy-alp-zseries.xml:46]:
```suggestion
          The deployment on the &zseries; architecture is specific and requires
```

**Review by dariavladykina — COMMENTED** [review]
Hi, please find some suggestions. Thanks!

File: tasks/deploy-alp-zseries.xml [hunk 7]
```diff
@@ -0,0 +1,245 @@
...
+        no interactive menu for configuring certain parameters of the
+        installation.
+      </para>
+    </note>
+  </section>
+  <section xml:id="requirements-deploy-alp-zseries">
+    <title>Requirements</title>
+    <para>
+      Before installing &productnameshort; on &zseries;, you need to fullfil
+      the following requirements:
```
[the remark this answers is not in the document]
Comment (tbazant) [reply tasks/deploy-alp-zseries.xml:41]: strange that the spell checker did not warn me :-(

**Pull request merged**